build-scripts/build_rule_playbooks.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+
+import argparse
+import ssg.playbook_builder
+
+
+def parse_args():
+    p = argparse.ArgumentParser()
+    p.add_argument(
+        "--input-dir", required=True, dest="input_dir",
+        help="Input directory that contains all Ansible remediations"
+        "snippets for the product we are building. "
+        "e.g. ~/scap-security-guide/build/fedora/fixes/ansible"
+    )
+    p.add_argument(
+        "--output-dir", required=True, dest="output_dir",
+        help="Output directory to which the rule-based Ansible playbooks "
+        "will be generated. "
+        "e.g. ~/scap-security-guide/build/fedora/playbooks"
+    )
+    p.add_argument(
+        "--product-yaml", required=True, dest="product_yaml",
+        help="YAML file with information about the product we are building. "
+        "e.g.: ~/scap-security-guide/rhel7/product.yml"
```

Comment:
We could transfer this to a positional argument, as it is the only argument that is essentially required.

Reply:
Depends on resolution of #4034